### PR TITLE
Supports exported snapshots for Pg 9.4+

### PR DESCRIPTION
**Pull request by cscetbon**
https://issues.jboss.org/browse/DBZ-1440

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@Naros, could you review this one? Thanks!

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java [hunk 1]
```diff
@@ -203,17 +203,16 @@ else if (shouldCreateSlot || !slotInfo.hasValidFlushedLsn()) {
     }
 
     private boolean useTemporarySlot() throws SQLException {
-        return dropSlotOnClose && isPg10OrGreater();
+        return dropSlotOnClose && isPg94OrGreater();
```
Comment (Naros) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java]: PostgreSQL did not add _Temporary Replication Slots_ until PG10, so I believe we need to keep the `isPg10OrGreater` method and revert this one line back to its original behavior.  @cscetbon can you make that change for me and run the test suite again to see if it fixes the problems on 9.6?

Comment (cscetbon) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java]: @Naros I sure can. And there is nothing else to do when it does not use temporary slots ?

Comment (Naros) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java]: Nope, that was the only issue that I saw.

**Comment by Naros** [comment]
Merged, thanks @cscetbon !

**Pull request merged**